app/src/main/java/com/example/myspacexdemoapp/ui/launch/holders/CategoryAdapter.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.myspacexdemoapp.ui.launch.holders
+
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+
+class CategoryAdapter : RecyclerView.Adapter<CategoryViewHolder>() {
```

Comment:
Why `CategoryAdapter` in `holders` package? It is better to rename to `adapter` and collect holders, adapters there